src/containers/PerfherderGraphContainer/index.jsx

```diff
@@ -48,8 +48,10 @@ class PerfherderGraphContainer extends Component {
           self.setState({ tooltipModel, canvas: this._chart.canvas });
         };
 
-        return { ...prevState, options, isLoading: false };
+        return { ...prevState, options };
       });
+    } finally {
+      this.setState({ isLoading: false });
     }
   }
 
```

Comment:
My mistake, setting `options.tooltips.custom`  should be done only if `getPerfherderData(series)` was successful, not in `finally{}`. :sweat_smile: